go.mod

```diff
@@ -2,7 +2,7 @@ module github.com/deepmap/oapi-codegen
 
 require (
 	github.com/cyberdelia/templates v0.0.0-20141128023046-ca7fffd4298c
-	github.com/getkin/kin-openapi v0.47.0
+	github.com/getkin/kin-openapi v0.49.0
```

Comment:
Could you rebase and use v0.53.0? The version you're using still forces Go 1.16 because kin-openapi has //embed directives.